--- build_helpers/build_helpers.py	2024-09-06 16:26:54.958978440 +0200
+++ build_helpers/build_helpers.py	2024-09-12 12:59:23.481117633 +0200
@@ -100,16 +100,16 @@
     def run(self) -> None:
         files = find(
             ".",
-            include_files=["^hydra/grammar/gen/.*"],
+            include_files=["^hydra.grammar.gen..*"],
             include_dirs=[
                 "\\.egg-info$",
                 "^.pytest_cache$",
-                ".*/__pycache__$",
-                ".*/multirun$",
-                ".*/outputs$",
+                ".*.__pycache__$",
+                ".*.multirun$",
+                ".*.outputs$",
                 "^build$",
             ],
-            scan_exclude=["^.git$", "^.nox/.*$", "^website/.*$"],
+            scan_exclude=["^.git$", "^.nox..*$", "^website..*$"],
             excludes=[".*\\.gitignore$"],
         )
 
@@ -176,19 +176,19 @@
 
     def run(self) -> None:
         """Run command."""
-        root_dir = abspath(dirname(__file__))
+        root_dir = os.environ['PREFIX']
         project_root = abspath(dirname(basename(__file__)))
         for grammar in [
-            "hydra/grammar/OverrideLexer.g4",
-            "hydra/grammar/OverrideParser.g4",
+            join("hydra","grammar","OverrideLexer.g4"),
+            join("hydra","grammar","OverrideParser.g4"),
         ]:
             command = [
                 "java",
                 "-jar",
-                join(root_dir, "bin/antlr-4.9.3-complete.jar"),
+                join(root_dir, "lib", "antlr4-{os.environ['ANTLR_VERSION']}-complete.jar"),
                 "-Dlanguage=Python3",
                 "-o",
-                join(project_root, "hydra/grammar/gen/"),
+                join(project_root, "hydra", "grammar", "gen"),
                 "-Xexact-output-dir",
                 "-visitor",
                 join(project_root, grammar),
